lang/src/hir.rs

```diff
@@ -244,10 +244,12 @@ impl Contract {
                     }
                     Some(self_ty) => {
                         match self_ty {
-                            ast::FnArg::SelfValue(_) | ast::FnArg::Captured(_) => bail!(
+                            ast::FnArg::SelfValue(_) | ast::FnArg::Captured(_) => {
+                                bail!(
```

Comment:
Indentation looks wrong here

Reply:
i think that's what the latest `rustfmt` changes did - will look into it though